docs/data/charts/components/components.md

```diff
@@ -87,6 +87,29 @@ For example, you can use `useLineSeries` to obtain the series of a line chart an
 
 {{"demo": "SeriesDemo.js"}}
 
+### Animation
+
+Some elements of charts are animated, such as the bars in a bar chart or the slices in a pie chart.
+
+We prioritize CSS animations, but some animations aren't possible using CSS. In those cases, we use JavaScript to animate elements.
+
+The animations of elements that are animated using CSS can be customized by overriding the CSS classes:
+
+{{"demo": "CSSAnimationCustomization.js"}}
+
+When it isn't possible to leverage CSS animations, the default components are animated using custom hooks.
+
+If you want to use the default animations in custom components, you can use these hooks.
+They are available for each element that is animated using JavaScript and are prefixed with `useAnimate`, e.g., `useAnimateLine` or `useAnimatePieArcLabel`.
```

Comment:
Maybe list all the `useAnimate` hooks instead? 🤔 

- `useAnimateLine`
- `useAnimatePieArcLabel`
- ...